app/controllers/supervisor/notes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Supervisor::NotesController < Supervisor::BaseController
+
+  def update
+    @notepad = @current_user.notepad
+    @notepad.update(notepad_params)
+    redirect_to supervisor_dashboard_path
+  end
```

Comment:
1. Currently if the given parameters are invalid the user is just redirected back to the dashboard, without knowing anything about the failure. 
2. Because the user is always redirected there's no need for the `@nodepad` instance variable, thus this could just be changed to `@current_user.notepad.update(nodepad_params)`.